File: codigo/soluciones/kat/bookingaroom.py
r, n = list(map(int, input().split()))
booked = [False for _ in range(r)]
for _ in range(n):
    x = int(input())
    booked[x - 1] = True
ans = False
for i in range(r):
    if booked[i]:
        continue
    print(i + 1)
    ans = True
    break
if not ans:
    print("too late")

File: codigo/soluciones/kat/bookingaroom.java
import java.io.*;
import java.util.*;
class Bookingaroom {
    private void _main() {
        Scanner sc = new Scanner(new BufferedInputStream(System.in));
        int r = sc.nextInt(), n = sc.nextInt();
        boolean[] booked = new boolean[r];
        for (int i = 0; i < r; ++i) {
            booked[i] = false;
        }
        while (n > 0) {
            --n;
            int aux = sc.nextInt();
            --aux;
            booked[aux] = true;
        }
        boolean ans = false;
        for (int i = 0; i < r; ++i) {
            if (booked[i]) continue;
            System.out.println(i + 1);
            ans = true;
            break;
        }
        if (!ans) {
            System.out.println("too late");
        }
        sc.close();
    }
    public static void main(String[] args) {
        (new Bookingaroom())._main();
    }
}
